package.json

```diff
@@ -16,7 +16,8 @@
     "nodemailer": "^6.2.1",
     "pg": "^7.9.0",
     "serve-favicon": "^2.5.0",
-    "uuid4": "^1.1.4"
+    "uuid4": "^1.1.4",
+    "youtube-audio-stream": "^0.3.2"
```

Comment:
Please let me know what is this new library for, if not necessary, don't push it onto our team branch and discard package-lock.json please